integration-cli/docker_cli_rename_test.go

```diff
@@ -19,12 +20,8 @@ func (s *DockerSuite) TestRenameStoppedContainer(c *check.C) {
 	dockerCmd(c, "rename", "first_name", newName)
 
 	name, err = inspectField(cleanedContainerID, "Name")
-	if err != nil {
-		c.Fatal(err)
-	}
-	if name != "/"+newName {
-		c.Fatal("Failed to rename container ", name)
-	}
+	c.Assert(err, check.IsNil, check.Commentf(name))
+	c.Assert(name != "/"+newName, check.Equals, false, check.Commentf("Failed to rename container %s", name))
```

Comment:
Woah this is funky to read.
How about `c.Assert(name, check.Equals, "/"+newName)`